regression/cbmc/xml-trace/test.desc

```diff
@@ -7,6 +7,7 @@ VERIFICATION FAILED
 <assignment assignment_type="state" base_name="u" display_name="test::u" hidden="false" identifier="test::u" mode="C" step_nr="\d+" thread="0">
 <location file=".*" function="test" line="\d+" working-directory=".*"/>
 <type>union myunion</type>
+<type>signed long( long)? int</type>
```

Comment:
Two `type` entries look confusing.

Reply:
The updated test.desc file now uses multi-line matching to clarify this.